aggregator/pom.xml

```diff
@@ -54,7 +54,7 @@
         <dependency>
             <groupId>io.quarkiverse.logging.cloudwatch</groupId>
             <artifactId>quarkus-logging-cloudwatch</artifactId>
-            <version>${cloudwatch.version}</version>
+            <version>${quarkkus-logging-cloudwatch.version}</version>
```

Comment:
```suggestion
            <version>${quarkus-logging-cloudwatch.version}</version>
```

Reply:
#protip: dont code before the first coffee. Fingers were a bit nervous obviously